keyboards/handwired/tsubasa/info.json

```
@@ -0,0 +1,75 @@
+{
+    "keyboard_name": "tsubasa",
+    "url": "https://github.com/kuriatsu/TSUBASA",
+    "maintainer": "kuriatsu",
```

You need the manufacturer name here.
```suggestion
    "maintainer": "kuriatsu",
    "manufacturer": "kuriatsu",
```